website/docs/introduction.md

```diff
@@ -9,8 +9,11 @@ Welcome to the JBrowse 2 documentation.
 
 We provide
 
-- [Quick start guide](quickstart_web) - for admins setting up JBrowse 2 on
-  their website
+- [Super-quick start guide](quickstart_web) - super brief setup instructions
+  e.g. 10 lines of CLI commands to help admins who are familiar with jbrowse
+  with getting setup
+- [Quick start guide](quickstart_web) - a more thourough quick-start guide to
+  help admins setting up JBrowse 2 on their website
```

Comment:
looks like these two links go to the same place

Reply:
nice catch, should be fixed